docs/architecture-overview.md

```diff
@@ -0,0 +1,173 @@
+
+All mobile app relevant files will be stored on the S3. Those files are:
+-	aggregated files containing the diagnostic keys, which were reported in a specific interval (e.g. hourly)
+-	Daily aggregated files containing the exposure keys, which were reported for this day.
+-	Configuration files, containing the [exposure configuration](https://developer.apple.com/documentation/exposurenotification/enexposureconfiguration) and CWA mobile app config
+-	Additional files regarding meta information of available files/structures/etc.
+
+The files will be pushed to an S3 compliant object store whenever new files become available. File structure definitions for those files can be found in the respective Services chapter.
+
+The mobile application will use a CDN for fetching files, which mirrors all files as a transparent proxy present in the object store.
+
+### Verification Backend
+
+Note: The verification back end design is not yet available as discussions are ongoing.
+
+The verification back end supports the user's journey beginning at scanning the QR code printed
+on the COVID-19 test strip until the upload of diagnosis keys when the user was tested positive. Testing
+labs will update the Verification Backend when a COVID-19 test result is positive. Since the QR code
+is linked to a test, the mobile application will be able to fetch a test-positive notification from
+the Verification Backend, along with a TAN. The TAN will be used as an authorization token when
+the user uploads the diagnosis keys of the past 14 days.
+
+Therefore, from a CWA Backend perspective, the Verification Backend provides an endpoint for TAN verification.
+
+## Security
+
+### Endpoint Protection
+
+The CWA back end exposes only one endpoint – the submission endpoint.
+The endpoint is public (unauthenticated), and authorization for calls is granted to users who are passing a valid TAN.
+The TAN verification cannot be done on CWA back end, but the task is delegated to the Verification Backend (see Verification Backend chapter in Integration with other Systems).
+
+### Authenticity
+
+All by CWA published files will be digitally signed by CWA.
+This ensures that clients can trust the file they have received from the third-party CDN.
+
+The proto files from Google/Apple already specify how signing should work. Each diagnosis key aggregate file
+will be a zip file, containing two files - one carries the actual payload. The other one carries signature information.
+
+CWA will be required to share the public key with Google/Apple, so that the API on the mobile client
+will be able verify against it.
+
+### Fake Submissions
+
+In order to protect the privacy of the users, the mobile app needs to send fake submissions from time to time.
+The server accepts the incoming calls and will treat them the same way as regular submissions.
+The payload & behavior of fake and real requests must be similar, so that 3rd parties are unable to differentiate between those requests.
```

Comment:
```suggestion
The payload and behavior of fake and real requests must be similar, so that 3rd parties are unable to differentiate between those requests.
```